CHANGELOG.md

```diff
@@ -3,18 +3,6 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
-# [1.5.0](https://github.com/adobe/spectrum-web-components/compare/v1.4.0...v1.5.0) (2025-04-15)
```

Comment:
Is this in here because the branch needs to be rebased with main?

Reply:
This is just for test. Once the PR is approved I will remove this generation.

Reply:
Yes, I was testing the new script.